docs/griptape-framework/drivers/structure-run-drivers.md

```diff
@@ -0,0 +1,97 @@
+## Overview
+Structure Run Drivers can be used to run Griptape Structures in a variety of runtime environments.
+When combined with the [Structure Run Task](../../griptape-framework/structures/tasks.md#structure-run-task) or [Structure Run Client](../../griptape-tools/official-tools/structure-run-client.md) you can create complex, multi-agent pipelines that span multiple runtime environments.
+
+## Local Structure Run Driver
+
+The [LocalStructureRunDriver](../../reference/griptape/drivers/structure_run/local_structure_run_driver.md) is used to run Griptape Structures in the same runtime environment as the code that is running the Structure.
+
+```python
+from griptape.drivers import LocalStructureRunDriver
+from griptape.rules import Rule
+from griptape.structures import Agent, Pipeline
+from griptape.tasks import StructureRunTask
+
+joke_teller = Agent(
+    rules=[
+        Rule(
+            value="You are very funny.",
+        )
+    ],
+)
+
+joke_rewriter = Agent(
+    rules=[
+        Rule(
+            value="You are the editor of a joke book. But you only speak in riddles",
+        )
+    ],
+)
+
+joke_coordinator = Pipeline(
+    tasks=[
+        StructureRunTask(
+            driver=LocalStructureRunDriver(
+                structure_factory_fn=lambda: joke_teller,
```

Comment:
Should this one also return a new instance, for similar reasons to my other comment?